README.md

```diff
@@ -285,3 +280,18 @@ PUPPET_VERSION=3.8.7 bundle package; mv Gemfile.lock Gemfile.lock.3.8.7
 PUPPET_VERSION=4.10 bundle package; mv Gemfile.lock Gemfile.lock.4.10
 PUPPET_VERSION=5.0 bundle package; mv Gemfile.lock Gemfile.lock.5.0
 ```
+
+## Maintainers
+
+* StackStorm <info@stackstorm.com>
+* James Fryman
+* Patrick Hoolboom
+* Nick Maludy [@nmaludy](https://github.com/nmaludy) <nick.maludy@encore.tech>
+
+## Help
```

Comment:
We can also place a
```
[![Join our community Slack](https://stackstorm-community.herokuapp.com/badge.svg)](https://stackstorm.com/community-signup)
```
in top of the README, so it'll show up as a badge, similar to StackStorm/st2 repo.